<!-- Run all nodes required for autonomous mode  -->
<launch>

    <node name="ImageProcessing" 
          pkg="cic" 
          type="ImageProcessing">
    <!-- Set Parameters for Image Processing node -->
    <param name="calibration_mode" value="false" type="bool" />
    <param name="pixel_cm_ratio_x" value="4" type="int" />
    <param name="pixel_cm_ratio_y" value="6" type="int" />
    <param name="scale_x" value="0.4" type="double" />
    <param name="scale_y" value="1.2" type="double" />
    <param name="p1" value="45" type="int" />
    <param name="p2" value="50" type="int" />
    <param name="p3" value="45" type="int" />
    <param name="p4" value="50" type="int" />
    </node>

    <node name="LaneDetection" 
          pkg="cic" 
          type="LaneDetection">
    <!-- Set Parameters for Image Processing node -->
    <param name="debug_mode" value="true" type="bool" />
    <param name="direct_mode" value="false" type="bool" />
    <param name="max_vel" value="-1000" type="int" />
    <param name="lane_width" value="110" type="int" />
    <param name="servo_center_position" value="90" type="int" />
    <param name="servo_step" value="2" type="int" />
    <param name="drive_right_lane" value="true" type="bool" />
    <param name="max_steering_angle_right" value="160" type="int" />
    <param name="max_steering_angle_left" value="20" type="int" />
    </node>

</launch>